sqlglot/parser.py

```diff
@@ -582,12 +582,37 @@ class Parser(metaclass=_Parser):
     }
 
     CONSTRAINT_PARSERS = {
-        TokenType.CHECK: lambda self: self.expression(
-            exp.Check, this=self._parse_wrapped(self._parse_conjunction)
+        "AUTOINCREMENT": lambda self: self._parse_auto_increment(),
```

Comment:
all of these can be changed to : self._parse_auto_increment

Reply:
We need the lambdas because we can't reference `self` there, right?

Reply:
ah right